autogpt_platform/backend/backend/util/test.py

```diff
@@ -81,7 +81,7 @@ async def is_execution_completed():
             graph_exec = await AgentServer().test_get_graph_run_results(
                 graph_id, graph_exec_id, user_id
             )
-            return graph_exec.node_executions
+            return graph_exec.node_executions  # type: ignore
```

Comment:
What happened here?

Reply:
`graph_exec.node_executions` isn't set if `graph_exec.user_id != graph.user_id`. This isn't relevant for any (current) test case though so we can ignore.
(I've added a comment.)

I could rewrite `test_get_graph_run_results` so that it always returns the node executions regardless of ownership, but it didn't seem worth the effort at the time.

Reply:
as it turns out, this did cause issues, so I rewrote the call chain